### PR TITLE
add note about User Assignment setting

### PR DESCRIPTION
**Pull request by lbk-fishtown**
## Description & motivation
"user assignment required" setting is an important aspect in Azure AD for controlling access to SAML apps

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Comment by lbk-fishtown** [comment]
If this setting is toggled to No then anyone with a link to dbt Cloud who is a member of Azure AD can sign in. This was picked up upon by one of our customers and they requested we update our doco.

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md [hunk 1]
```diff
@@ -92,6 +92,9 @@ Once you've registered the application, the next step is to assign users to it.
 
 <Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-enterprise-app-users.png" title="Adding Users to an Enterprise Application a Redirect URI"/>
 
+:::info User assignment required?
+Under **Properties** check the toggle setting for **User assignment required?** and confirm it aligns to your requirements. Most customers will want this toggled to **Yes** so that only users/groups explicitly assigned to dbt Cloud will be able to sign in. If this setting is toggled to **No** any user will be able to access the application if they have a direct link to the application per [Azure AD Documentation](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/assign-user-or-group-access-portal#configure-an-application-to-require-user-assignment) 
+
```
Comment (annafil) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:98]:
```suggestion
:::
```

Think this needs to be closed to render correctly :)

Comment (lbk-fishtown) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:98]: Thank you for catching that!

Comment (lbk-fishtown) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:98]: Committed update to resolve this.

**Review by annafil — APPROVED** [review]

**Pull request merged**